src/nw4r/db/db_directPrint.cpp

```diff
@@ -52,7 +52,7 @@ static const u32 sFontData2[77] = {
     0x100FBE1C, 0x10089008, 0x60070808, 0x00000000, 0x02000200, 0x7A078270, 0x8BC81E88, 0x8A2822F8,
     0x9A282280, 0x6BC79E78, 0x30000000, 0x48080810, 0x41E80000, 0x422F1830, 0xFBE88810, 0x40288890,
     0x43C89C60, 0x81000000, 0x81000000, 0x990F3C70, 0xA10AA288, 0xE10AA288, 0xA10AA288, 0x98CAA270,
-    0x00000000, 0x00000020, 0xF1EF1E20, 0x8A28A0F8, 0x8A281C20, 0xF1E80220, 0x00000000, 0x00000000,
+    0x00000000, 0x00000020, 0xF1EF1E20, 0x8A28A0F8, 0x8A281C20, 0xF1E80220, 0x80283C38, 0x00000000,
```

Comment:
What was the reason for this change? it seems its not correctly byte matched?
![image](https://github.com/lepelog/ss-dtk/assets/42302100/6fc076e4-d788-4d7a-a052-5ac0610deab2)

Reply:
Oh interesting. It just isnt correctly shown in objdiff, my bad.